08_flask_html_basics.md

```diff
@@ -0,0 +1,601 @@
+
+For example, `https://python-intro.devops.corndel.com/08_flask_html_basics.html` is a URL composed of the following:
+
+- A _protocol_: In this case it's `https` (HyperText Transfer Procol Secure), but others exist such as `http`, `ftp`, etc. This establishes some standardised rules of communication between the application and the browser.
+- A _hostname_: In this case it's `python-intro.devops.corndel.com`. This refers to the domain name of the website or application that you're visiting.
+- A _resource_ or _file name_: In this case it's `08_flask_html_basics.html`, which refers to the HTML file used to display this page.
+
+## What is HTML?
+
+_HTML_ stands for HyperText Markup Language. As its name suggests, HTML is a Markup Language, not a programming language such as python. A Markup Language refers to a system used for encoding text, which allows inserting special characters to structure the text and alter the way it is displayed.
+
+### Exercise 8.1
+
+Let's try to create a simple web page. You can do that by creating a new file - the name is not important, but the extension must be `.html`. Within that file, you can just type in `Hello World!` and save.
+
+Now, if you open the file using any browser, you should see the text `Hello World!` written in black text on a white background.
+
+## HTML Elements and tags
+
+Since HTML is a markup language, it is possible to alter the way the text is structured and displayed. This is done through the use of `elements` and `tags`.
+
+You can think of an HTML page as a document, which is composed of several elements. Tags are used to mark the beginning and end of an element, and are usually pre-defined keywords enclosed in angle brackets. To denote the start of an element, you would see a tag such as `<tag>`, and to mark where the element ends, the tag is accompanied by a forward slash before the keyword, such as `</tag>`.
+
+> Many elements allow other elements to be nested inside them (referred to as _child nodes_). Child nodes refer to those above them as _parent nodes_.
+
+> Tags are not displayed by the browser. Instead, they instruct the browser on _how_ to display the HTML element.
+
+There are also elements that do not have an end tag and cannot contain child (nested) nodes. These are called [void elements](https://developer.mozilla.org/en-US/docs/Glossary/Void_element), and among those we will use `<input>` and `<img>`.
+
+As an example:
+- `<html>` and `</html>` are tags
+- `<html> Hello World! </html>` is an element
+- `<input>` is also an element
+
+### Exercise 8.2
+
+Let's inspect how the browser views our document. Right click anywhere on the page and then pick "Inspect" (this can have a different but similar name depending on your browser, such as "Inspect Element" or "Inspector"). This should open up a panel on the right side. Navigate to the `Elements` tab, where you will see the `Hello World!` text surrounded by some tags.
```

Comment:
```suggestion
Let's inspect how the browser views our document. Right click anywhere on the page and then pick "Inspect" (this can have a different but similar name depending on your browser, such as "Inspect Element" or "Inspect"). This should open up a panel on the right side. Navigate to the `Elements` (or `Inspector`) tab, where you will see the `Hello World!` text surrounded by some tags.
```

Reply:
I've only added the second suggested change, because "Inspector" was already there and it would've caused a repetition